_config.yml

```diff
@@ -1,25 +1,12 @@
 ### Site info:
 title: SEMBA
 tagline: EMC BroadBand simulator
-#author: Lijia Yu
```

Comment:
How are used these lines? Research if those can be used for something useful. Then it can be documented and the client informed.

Reply:
I see it in this ticket: https://aircuryinterns.youtrack.cloud/issue/SH-12/Variables-investigation. It wasn't needed a new one for this but it's ok